sources/platform/integrations/pinecone.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Pinecone
+description: Learn how to integrate Apify with Pinecone in order to feed data crawled from the web into the Pinecone vector database.
+sidebar_position: 11.22
+slug: /integrations/pinecone
+---
+
+**Learn how to integrate Apify with Pinecone to feed data crawled from the web into the Pinecone vector database.**
+
+---
+
+> For more information on Pinecone, visit its [documentation](https://docs.pinecone.io).
+
+## What is Pinecone?
+
+Pinecone is a managed vector database that allows users to store and query dense vectors for AI applications such as recommendation systems, semantic search, and retrieval augmented generation.
+
+## How to integrate Apify with Pinecone?
+
+This example focuses on integrating Apify with Pinecone using the Apify Actor.
```

Comment:
## How to integrate Apify with Pinecone?

The Apify integration for Pinecone lets you export results from Actors and Dataset items into a specific Pinecone vector index. Note that you also need an [OpenAI platform](https://platform.openai.com/docs/overview) account with an API key in order to compute the text embeddings.

The integration is facilitated using the [Pinecone integration](https://apify.com/jan.turon/pinecone-integration) Actor.

## Examples

These examples use the [Website Content Crawler](https://apify.com/apify/website-content-crawler), an Actor that can deeply crawl websites, clean their HTML by removing a cookies modal, footer, or navigation, and then transform the HTML to Markdown, which can be used as training data for AI models or for feeding LLM and generative AI applications with web content.

### User interface

...

### Python

....

Reply:
Thanks, added.